### PR TITLE
Adding metadata for vaadin doc site

### PR DESCRIPTION
**Pull request by manolo**
closes vaadin/components-team-tasks#6

[Review on Reviewable](https://reviewable.io/reviews/vaadin/vaadin-grid/186)

### CODE REVIEW

**Comment by tomivirkki** [comment]
Reviewed 14 of 14 files at r1.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-grid/186)

File: demo/formatting.html [hunk 1]
```diff
@@ -1,4 +1,9 @@
 <!doctype html>
+<!--
+title: Formating Data
```
Comment (jounik) [inline demo/formatting.html]: Typo, should be "Formatting"

File: demo/editor.html [hunk 2]
```diff
@@ -1,4 +1,9 @@
 <!doctype html>
+<!--
+title: Grid Editor
```
Comment (jounik) [inline demo/editor.html]: This should not be listed in the docs site, because it’s not available. Could be removed completely from the repo.

**Comment by manolo** [comment]
@jounik fixed : https://github.com/vaadin/vaadin-grid/commit/231aeb14b97ab277ecd9bc33754766b5e07d1602

**Comment by jounik** [comment]
@manolo great, thanks!

**Pull request merged**